redwood-compose/src/commonMain/kotlin/app/cash/redwood/compose/WidgetApplier.kt

```diff
@@ -66,6 +66,13 @@ internal class NodeApplier<W : Any>(
     }
   }
 
+  override fun onBeginChanges() {
```

Comment:
Does this defer changes by 16ms?

Reply:
Since the callback is now called earlier, it means you have to rely on the frame ending to know when to send changes. Since most of the time there's only a single set of changes performed to the applier, it's mostly a no-op.